frontend/components/metric-progress-circular.vue

```diff
@@ -35,5 +35,8 @@ const { label, value, size, width } = withDefaults(defineProps<Props>(), {
   width: 20,
 });
 
-const progressValue = computed(() => typeof value === "number" ? clamp(value, { min: 0, max: 100 }) : 0);
+const progressValue = computed(() => typeof value === "number"
+  ? clamp(value, { min: 0,
+    max: 100 })
```

Comment:
This, along with a few other things in this PR, seem to have weird formatting. I think most of your linting changes are good additions, but allowing this newline inconsistency seems _wrong_.